pull_request_template.md

```diff
@@ -0,0 +1,18 @@
+**Solution Descrition:**
+
+A clear and concise solution description of the problem.
```

Comment:
May want to reword as "A clear description of the problem and solution".

Reply:
sure